vuepress/docs/v6.3/docs/getting-started/README.md

```diff
@@ -205,9 +205,8 @@ database connection.
 
 #### Configure Access to Your Cluster
 
-::: tip
-Entando sets up `Ingresses` in Kubernetes to access services from outside your server cluster.
-We'll use this to access Entando from a local browser. 
+::: tip What about networking?
```

Comment:
Capitalize to match others or lowercase others

Reply:
will do